.travis.yml

```diff
@@ -15,6 +15,10 @@ env:
   - TEST_DIR=/tmp/test_dir/
   - MODULE=openml
   matrix:
+  - DISTRIB="conda" PYTHON_VERSION="3.7" SKLEARN_VERSION="0.23.1" TEST_DIST="true"
```

Comment:
Would it be possible that you sort these a bit better? I would suggest that we sort them by scikit-learn version, and then by Python version; and have the coverage and flake tests at the top.

Reply:
Done.